### PR TITLE
refector: change deprecated "terminal" to "term"

### PR DESCRIPTION
**Pull request by yj-yoo**
### What this PR does
Change deprecated "golang.org/x/crypto/ssh/terminal" to "golang.org/x/term"
reference: https://pkg.go.dev/golang.org/x/crypto/ssh/terminal

Fixes 

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

**Comment by oshoval** [comment]
Thanks

Note please that the ref issue includes more action items and will be closed once this PR is merged
so not sure if it is correct to ref it as fixed

/ok-to-test

File: pkg/virtctl/utils/utils.go [hunk 1]
```diff
@@ -2,11 +2,12 @@ package utils
 
 import (
 	"fmt"
+
+	"golang.org/x/term"
```
Comment (oshoval) [inline pkg/virtctl/utils/utils.go:6]: can you put this please as the last group ?
also the fmt should be still in the group it was

**Comment by yj-yoo** [comment]
@oshoval 
There was an opinion that PR had become too big to solve many problems and that it would be good to divide it. Should I remove the link from the ref issue first?

**Comment by oshoval** [comment]
> @oshoval There was an opinion that PR had become too big to solve many problems and that it would be good to divide it. Should I remove the link from the ref issue first?

yes please i think it is better
once all the action items are closed we can close the issue manually (we can just ref the issue on the relevant PRs
without "Fixes")

File: pkg/virtctl/utils/utils.go (hunk 1 above)
Comment (yj-yoo) [reply pkg/virtctl/utils/utils.go]: @oshoval 
okay. Applied!
I think this project needs to be careful about using "make generate"...

Comment (oshoval) [reply pkg/virtctl/utils/utils.go]: Thanks
we have `pull-kubevirt-generate`
and without the right updated bazel files, it fails compilation

**Comment by oshoval** [comment]
It worth please to squash the commits in this case

**Comment by yj-yoo** [comment]
@oshoval Thanks your advice☺️

**Comment by oshoval** [comment]
/ok-to-test

Thanks
lets just see all pass and then it is lgtm

**Comment by oshoval** [comment]
did we remove /ok-to-test ?

/lgtm

Thanks

**Comment by oshoval** [comment]
Thanks
/lgtm

**Comment by yj-yoo** [comment]
@oshoval 
While I was resolving other issues in this project, an error occurred in 'pull-kubevirt-verify-go-mod' after RP. I took a hint from your advice and ran '/make deps-sync', but an error occurred in go mod vendor. Can I get a hint?

**Comment by oshoval** [comment]
> @oshoval While I was resolving other issues in this project, an error occurred in 'pull-kubevirt-verify-go-mod' after RP. I took a hint from your advice and ran '/make deps-sync', but an error occurred in go mod vendor. Can I get a hint?

is the error you see is as in this PR 'pull-kubevirt-verify-go-mod' ?
https://github.com/kubevirt/kubevirt/pull/11348#issuecomment-1970793410
if so it is something global, not just on the PR

**Comment by yj-yoo** [comment]
@oshoval 
Oh... that's right... 'pull-kubevirt-verify-go-mod' is the same as the local command result...

Then, it's clearly a problem with 'pull-kubevirt-unit-test-arm64'... Actually, I'm not familiar with testing for this project, so I'm having a hard time... Thank you, I'll do more research!

p.s. The test for this project just now failed... I'm scared of this place now...😭

**Comment by oshoval** [comment]
> @oshoval Oh... that's right... 'pull-kubevirt-verify-go-mod' is the same as the local command result...
> 
> Then, it's clearly a problem with 'pull-kubevirt-unit-test-arm64'... Actually, I'm not familiar with testing for this project, so I'm having a hard time... Thank you, I'll do more research!
> 
> p.s. The test for this project just now failed... I'm scared of this place now...😭

Don't be scared, and you don't need to handle it all, of course when it is a global problem
Thank you for your contributions

**Comment by yj-yoo** [comment]
@oshoval 
Thanks for the encouragement. The project's reviewer just said that an error occurred, so I have to go back to studying!

**Comment by oshoval** [comment]
Do you mean the failure here ?
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11351/pull-kubevirt-goveralls/1763174958747357184

It is just bootstrap problem, it is not related to the PR
We can wait until `pull-kubevirt-goveralls` is globally fixed and then continue as it isn't related to this PR

**Comment by yj-yoo** [comment]
@oshoval 
Sorry for confusing you. I mentioned two failures in this conversation.

1. The error you mentioned is the one I mentioned in the previous conversation p.s. This is the error mentioned in . You're helping me so well here, so I don't have to worry. https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11351/pull-kubevirt-goveralls/1763174958747357184

2. Another error I need to study is #11348. I guess I'll have to try until someone gives me a hint.
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11348/pull-kubevirt-unit-test-arm64/1762871293239103488

**Comment by oshoval** [comment]
About [2]
please see
https://github.com/kubevirt/kubevirt/pull/11321#issuecomment-1957064347
It is a flake, not related to your PR, so not need to be addressed on your PR, unless you want to :)
Thanks

**Comment by oshoval** [comment]
sure thanks
once you doing a chance, please add information on what you did, it helps and also
will triggers the reviewers to review again

/lgtm

**Review by xpivarc — COMMENTED** [review]
/approve

**Pull request merged**